src/Symfony/Component/Intl/Tests/TimezonesTest.php

```diff
@@ -595,8 +604,10 @@ public function testGetCountryCodeWithUnknownTimezone()
      */
     public function testGetGmtOffsetAvailability(string $timezone)
     {
-        $this->assertInternalType('int', Timezones::getRawOffset($timezone));
-        $this->assertInternalType('string', Timezones::getGmtOffset($timezone));
+        Timezones::getRawOffset($timezone);
+        Timezones::getGmtOffset($timezone);
+
+        $this->addToAssertionCount(1);
```

Comment:
This looks weird. What's the point of such test?

Reply:
it ensures for each timezone we have offset (raw+labeled).

It's ensured by return-type, so if something unexpected happens it violates and this test fails. It's just to notify us if we ever add a timezone with inconsistent/incomplete data.

Reply:
Maybe we can add a comment on each such assertions to explain the goal?

Reply:
done.